_sass/blocks/_revenues_subpage_intro.scss

```diff
@@ -3,32 +3,58 @@
 // This is the first block on a revenues subpage
 //
 // Styleguide blocks.revenues_subpage_intro
+.revenues_subpage-intro_wrapper {
+  border-top: 1px solid $mid-gray;
+}
 
 .revenues_subpage-intro {
-  padding-top: $base-padding;
+  padding-top: $base-padding-extra;
 
   h1 {
-    @include heading(h1);
-    padding-bottom: ($base-padding / 2);
+    margin-bottom: 0.7rem;
+    line-height: 2rem;
   }
 
-  p {
-    padding-bottom: $base-padding;
-  }
+}
+
+.revenues_subpage-intro_text{
+  @include heading('para-lg');
+  padding-bottom: $base-padding;
+}
+
+.revenues_subpage-breadcrumb {
+  color: $dark-gray;
+  @include heading('para-lg');
 
 }
 
 .revenues_subpage-office {
   background-color: $white;
-  border: $base-border-size solid $mid-gray;
-  border-radius: $base-border-radius;
-  margin-bottom: $base-padding;
-  padding-top: $base-padding;
+  &:first-child {
+    padding-top: $base-padding;
```

Comment:
Properties should be ordered border-top-left-radius, border-top-right-radius, padding-top